### PR TITLE
finished version

### PR DESCRIPTION
**Pull request by MattLjunggren**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,139 @@
...
+		Your video does not support the video tag
+	  </video>
+</div>
+
+	</Header> 
+
+	<main>		
+		
+		<!-- Signup form -->
+		<section class="wrapperSignIn">
```
Comment (joannalodell19) [inline code/index.html]: You might want to take a look at the indentation styling. This is a website I've found helpful: [https://granneman.com/webdev/coding/formatting-and-indenting-your-html](url)

File: code/script.js [hunk 2]
```diff
@@ -5,5 +5,12 @@ function toggle() {
   this.classList.toggle("active")
```
Comment (joannalodell19) [inline code/script.js:5]: Nice work with the JavaScript!

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,249 @@
...
+	justify-content: space-around;
+	margin: 30px 20% 50px 20%;
+
+}
+
+.checkbox {
+	font-size: 1.5rem;
+}
+
+input[type="radio"] {
```
Comment (joannalodell19) [inline code/style.css]: I learned something new here regarding radio buttons and check-boxes, haven't used them yet. Thanks for that. :)

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,249 @@
...
+	margin: 40px auto;
+	font-size: 1.8rem;
+	border-radius: 10px;
+}
+
+input[type="checkbox"] {
+	transform: scale(1.3);
+}
+
+/* End of Sign in form */
```
Comment (joannalodell19) [inline code/style.css]: Nice done with the comments here. Easy to follow!

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,249 @@
...
+	}
+	
+
+}
+/* End of Accordion */
+
+/* Tablet size */
+@media (min-width: 668px) {
+	#headerWrapper {
+		width: 1100px;
```
Comment (joannalodell19) [inline code/style.css]: Could this be why the video isn't responsive for bigger screens? That there is a width to it of 1100px. And not a styling for it in the media query for desktop?

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,249 @@
...
+		justify-content: space-around;
+		margin: 30px 20% 50px 20%;
+	
+	}
+}
+
+/* Desktop size */
+
+@media (min-width: 1025px) {
+
```
Comment (joannalodell19) [inline code/style.css]: So maybe input styling for the video for devices with a width over 1100px here?

**Review by joannalodell19 — CHANGES_REQUESTED** [review]
Good job! I can really see what you were going for. I like the idea and concept even though the site wasn’t responsive for desktop. 

Your code was easy to follow due to your comments! 

It seems to me like you were really close to making it work as you wanted to!

File: code/style.css (hunk 5 above)
Comment (MattLjunggren) [reply code/style.css]: Det stämde, tackar!

File: code/index.html (hunk 1 above)
Comment (MattLjunggren) [reply code/index.html]: I VSC så är det rätt indenterat, oklart varför det ändrar sig här?

**Pull request closed without merging**